cypress/support/pages/login_management/pageLoginManagement.js

```diff
@@ -183,17 +186,54 @@ class Login_Management {
 	}
 
 	clickOnForgotPassword() {
-		cy.get(Login_Management.#passwordRecoveryButton).click();
+		cy.get("body").then((body) => {
+			if (body.find(Login_Management.#buttonLoginViaEmailNbc).length > 0) {
+				// If it exists in case of NBC login page, click on it and then the second element
+				cy.get(Login_Management.#buttonLoginViaEmailNbc)
+					.first()
+					.click()
+					.then(() => {
+						// locate the button in the specific class as they are used in multiple places
+						const pwdRecoveryButtonSelector =
```

Comment:
why we have to create this selector, if there are multiple selector then it would either be the first of the last one? it would be better to use that API's here

Reply:
I updated the code differently now, class is a parent element here.